lib/ansible/modules/network/meraki/meraki_static_route.py

```diff
@@ -0,0 +1,243 @@
+    returned: info
+    type: complex
+    contains:
+      net_id:
+        description: Identification string of network.
+        returned: success
+        type: string
+        sample: N_12345
+      net_name:
+        description: Written name of network.
+        returned: success
+        type: string
+        sample: YourNet
+      name:
+        description: Name of static route.
+'''
+
+import os
+from ansible.module_utils.basic import AnsibleModule, json, env_fallback
+from ansible.module_utils.urls import fetch_url
+from ansible.module_utils._text import to_native
+from ansible.module_utils.network.meraki.meraki import MerakiModule, meraki_argument_spec
+
+
+def get_static_routes(meraki, net_id):
+    path = meraki.construct_path('get_all', net_id=net_id)
+    r = meraki.request(path, method='GET')
+    return r
+
+
+def get_static_route(meraki, net_id, route_id):
+    path = meraki.construct_path('get_one', net_id=net_id, custom={'route_id': meraki.params['route_id']})
+    r = meraki.request(path, method='GET')
+    return r
+
+
+def main():
+
+    # define the available arguments/parameters that a user can pass to
+    # the module
+
+    argument_spec = meraki_argument_spec()
+    argument_spec.update(
+        net_id=dict(type='str'),
+        net_name=dict(type='str'),
+        name=dict(type='str'),
+        subnet=dict(type='str'),
+        gateway_ip=dict(type='str'),
+        state=dict(type='str', choices=['absent', 'query', 'present'], default='present'),
+        fixed_ip_assignments=dict(type='list'),
+        reserved_ip_ranges=dict(type='list'),
+        route_id=dict(type='str'),
+    )
+
+    # the AnsibleModule object will be our abstraction working with Ansible
+    # this includes instantiation, a couple of common attr would be the
+    # args/params passed to the execution, as well as if the module
+    # supports check mode
+    module = AnsibleModule(argument_spec=argument_spec,
+                           supports_check_mode=False,
+                           )
+
+    meraki = MerakiModule(module, function='static_route')
+    module.params['follow_redirects'] = 'all'
+    payload = None
+
+    query_urls = {'static_route': '/networks/{net_id}/staticRoutes'}
+    query_one_urls = {'static_route': '/networks/{net_id}/staticRoutes/{route_id}'}
+    create_urls = {'static_route': '/networks/{net_id}/staticRoutes/'}
+    update_urls = {'static_route': '/networks/{net_id}/staticRoutes/{route_id}'}
+    delete_urls = {'static_route': '/networks/{net_id}/staticRoutes/{route_id}'}
+    meraki.url_catalog['get_all'].update(query_urls)
+    meraki.url_catalog['get_one'].update(query_one_urls)
+    meraki.url_catalog['create'] = create_urls
+    meraki.url_catalog['update'] = update_urls
+    meraki.url_catalog['delete'] = delete_urls
+
+    if not meraki.params['org_name'] and not meraki.params['org_id']:
+        meraki.fail_json(msg='org_name or org_id parameters are required')
```

Comment:
We tend to quote names and values in messages for readability.
```suggestion
        meraki.fail_json(msg="Parameters 'org_name' or 'org_id' parameters are required")
```